src/ReactDOM.re

```diff
@@ -652,7 +652,7 @@ type domProps = {
   [@mel.optional]
   acceptCharset: option(string),
   [@mel.optional]
-  action: option(string), /* uri */
```

Comment:
This comment and the one in `rei` were useful imo. Or were they not accurate?

Reply:
@davesnx wasn't there something about this field that bothered you a bit? 

can you link us to the issue about this? I think something about `action` being a function for RSC or something?

Reply:
Reverting this commit.

`action` is problematic in React 19 since it overloads it. formAction and action in this PR they are sync (while in React can be async and have different behaviour), if users want to use it they would need to use createElement + cloneElement or one of those tricks.